### PR TITLE
Support build vagrant box in parallel

### PR DESCRIPTION
**Pull request by sharkconi**
Signed-off-by: Ma, Mark <Mark.Ma@emc.com>

### CODE REVIEW

**Comment by sharkconi** [comment]
@PaynePei  @xiar @fub2 @turtle-fly

File: virtualbox_builder/virtualbox_builder.sh [hunk 1]
```diff
@@ -76,10 +76,22 @@ $PARTED --align=none -s $IMAGE set 1 boot on > /dev/null 2>&1
...
-loop_dev=`losetup -f`
+#
+# find first available loop device
+#
+loop_dev=""
+loop_device_name=""
+while true
+do
+    loop_dev=`losetup -f`
+    [ -z "$loop_dev" ] && exit 1
```
Comment (PaynePei) [inline virtualbox_builder/virtualbox_builder.sh]: should add one error message?

**Comment by PaynePei** [comment]
so speedy fix, thanks for your efforts, @sharkconi

**Comment by xiar** [comment]
:+1:

**Pull request merged**